apps/condo/domains/contact/schema/Contact.js

```diff
@@ -96,14 +100,15 @@ const Contact = new GQLListSchema('Contact', {
     hooks: {
         validateInput: async ({ resolvedData, operation, existingItem, addValidationError }) => {
             const newItem = { ...existingItem, ...resolvedData }
-            const { property, unitName, name, phone } = newItem
+            const { property, unitName, unitType, name, phone } = newItem
 
             if (operation === 'create' && !property) {
                 return addValidationError(`${ PROPERTY_REQUIRED_ERROR }] no property for contact`)
             }
             const condition = {
                 property: { id: property },
                 unitName: unitName || null,
+                unitType,
```

Comment:
What will happen if we will try to create a contact, having another contact with ‘unitType’ with ‘null’ value, that was created before this changes in validation logic?

Reply:
> @AntonAL We won't have a case where `unitType` can be `null`, since all values will be overwritten by the default value via migration

@sitozzz, I don't see a default value setting in migration.
Take a look at:
```
ALTER TABLE "Contact" ALTER COLUMN "unitType" DROP DEFAULT;
```

Reply:
> > @AntonAL We won't have a case where `unitType` can be `null`, since all values will be overwritten by the default value via migration
> 
> @sitozzz, I don't see a default value setting in migration. Take a look at:
> 
> ```
> ALTER TABLE "Contact" ALTER COLUMN "unitType" DROP DEFAULT;
> ```

@AntonAL plz take a look at this line of code `ALTER TABLE "Contact" ADD COLUMN "unitType" text DEFAULT 'flat' NOT NULL;`